source/tools/timemory-avail/CMakeLists.txt

```diff
@@ -7,8 +7,15 @@ endif()
 add_executable(timemory-avail ${_EXCLUDE}
     ${CMAKE_CURRENT_LIST_DIR}/timemory-avail.cpp
     ${CMAKE_CURRENT_LIST_DIR}/timemory-avail.hpp)
+set_target_properties(timemory-avail PROPERTIES DEBUG_POSTFIX "${CMAKE_DEBUG_POSTFIX}")
```

Comment:
So I'd prefer this to just be removed. The build-type doesn't really matter for this particular executable since it just prints out information. Even if the debug version is installed over the release version, we are probably only talking about a difference on the order of tenths of a second to run the query.

Reply:
I agree and I have removed it. 

(But on windows keep in mind that the difference between debug and release can be much more than simply performance. Their dependencies are different, so one may work and the other may not work depending on what is in your path. Having the same utility executable for both debug and release configurations can help resolve environmental problems. I do not want to belabor the point, but just point out some idiosyncracies on windows that you do not have to deal with on linux.)